### PR TITLE
Basic integration for Azure Web PubSub Service

### PR DESCRIPTION
**Pull request by xingsy97**
Extension for [Azure Web PubSub service](https://azure.microsoft.com/en-us/products/web-pubsub) is under developing.
Add basic integration for it.

### CODE REVIEW

**Comment by xingsy97** [comment]
@microsoft/vscodeazuretoolsdev

File: .github/workflows/build-and-release.yml
Comment (alexweininger) [inline .github/workflows/build-and-release.yml:1]: What is this for? We already handle extension releases and I think this file should be removed.

File: .vscode/launch.json [hunk 1]
```diff
@@ -68,7 +68,8 @@
                 "--extensionDevelopmentPath=${workspaceFolder}/../vscode-cosmosdb",
                 "--extensionDevelopmentPath=${workspaceFolder}/../vscode-azurecontainerapps",
                 "--extensionDevelopmentPath=${workspaceFolder}/../vscode-azurespringcloud",
-                "--extensionDevelopmentPath=${workspaceFolder}/../vscode-azurelogicapps"
+                "--extensionDevelopmentPath=${workspaceFolder}/../vscode-azurelogicapps",
+                "--extensionDevelopmentPath=${workspaceFolder}/../vscode-azurewebpubsub",
```
Comment (alexweininger) [inline .vscode/launch.json:72]: This isn't required and can be removed. This is meant for our own testing of Resource Groups. If you want to test your extension with Resource Groups, add RGs to the launch.json of your extension.

File: package.json [hunk 2]
```diff
@@ -575,7 +575,7 @@
         "build": "tsc && gulp webpack-prod",
         "cleanReadme": "gulp cleanReadme",
         "compile": "tsc -watch",
-        "package": "vsce package --githubBranch main",
+        "package": "vsce package --githubBranch wps",
```
Comment (alexweininger) [inline package.json:578]: Revert this?

File: package.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
     "name": "vscode-azureresourcegroups",
     "displayName": "Azure Resources",
     "description": "%azureResourceGroups.description%",
-    "version": "0.8.1-alpha.0",
+    "version": "0.8.1-alpha.2",
```
Comment (alexweininger) [inline package.json:5]:
```suggestion
    "version": "0.8.1-alpha.0",
```

We'll bump the version once we're ready to release

**Comment by alexweininger** [comment]
Do you have a link to the source code of the extension?

**Comment by xingsy97** [comment]
@alexweininger All comments are resolved. I accidentally updated the branch for my personal test use several days ago.

**Comment by alexweininger** [comment]
> for the unsolved question I asked you whether the code could be put into our all-in-one repo https://github.com/azure/azure-webpubsub

It's fine if the code is located in an all-in-one repo. Our team probably prefers it to be stand alone, but it's fine either way. For example, the AZD extension is located in an all-in-one repo. https://github.com/Azure/azure-dev/tree/main/ext/vscode

**Review by alexweininger — APPROVED** [review]

**Pull request merged**